src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/User.java

```diff
@@ -51,6 +55,20 @@
 	)
 	private Set<Role> roles = new HashSet<Role>();
 
+	@ManyToMany
+	@JoinTable(
+		name = "user_usergroup",
```

Comment:
Givinig the JoinTable the same name leads to just one table. So the information is shared and not doubled.